survey_formio/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,8 @@
+This module adds compatibility bewtween form.io and Survey. It adds a function on survey.survey that generates
+a form.io compatible JSON that represents that survey. It also adds a function to generate a survey.user_input
+from a JSON returned by form.io.
+Use generate_formio_json to get the json representing a survey.
+Use user_input_from_formio to create a user_input with the answer from form.io
+
+As of now this export does not support validation options other than survey_question.constr_mandatory. It also does
+not work with question of type 'matrix'
```

Comment:
Move this to the roadmap section?

Reply:
I've moved the info about validation. I didn't move the second part, I've only reworked it a bit. I don't think it should be in the roadmap because I don't think it'll ever be supported as there's no corresponding component in formio.